pyproject.toml

```
@@ -47,7 +47,7 @@ tests = [
 doc = [
     "ansys_sphinx_theme[autoapi]==0.15.2",
     "numpydoc==1.7.0",
-    "Sphinx==7.2.6",
+    "Sphinx==7.3.0",
```

```suggestion
    "Sphinx==7.3.5",
```
I would advise going straight for 7.3.5 instead as multiple versions have been release one after another to fix multiple bugs. Seems like the current error in CICD is related to [pydata_sphinx_theme](https://www.sphinx-doc.org/en/master/changes.html#release-7-3-2-released-apr-17-2024)

Hopefully, using 7.3.5 will fix every issue we might have while upgrating sphinx.

---

Feel free to discard this comment if it does not make sense to handle things through this PR :)

---

https://www.sphinx-doc.org/en/master/changes.html 7.3.5 fixes the issue.